packages/bff/src/services/agreementService.ts

```diff
@@ -349,3 +459,26 @@ export function getCurrentDescriptor(
   }
   return descriptor;
 }
+
+function parseMediaType(
```

Comment:
I would change the name of this function into `assertContentMediaType` (or something like that) and move it into the `validators.ts` file, wdyt?

Reply:
Yes, that is indeed more descriptive, thanks for the suggestion!